app/components/AnalyzeComponent.js

```diff
@@ -17,15 +19,21 @@ import {
 } from '../constants/constants';
 import {
   readWorkspaceCleanedEEGData,
-  getSubjectNamesFromFiles
+  getSubjectNamesFromFiles,
+  readWorkspaceBehaviorData,
+  readBehaviouralData,
+  storeAggregatedBehavioralData,
```

Comment:
Should decide on whether we're using BehaviorData or Behaviour*al*Data

Reply:
I changed to BehaviorData